frontend/packages/dev-console/src/components/edit-deployment/EditDeploymentPage.tsx

```diff
@@ -16,8 +16,15 @@ const EditDeploymentPage: React.FC<EditDeploymentPageProps> = ({ match, location
   const queryParams = new URLSearchParams(location.search);
   const kind = queryParams.get('kind');
   const name = queryParams.get('name');
+  const kindModel = modelFor(kind) ?? {};
```

Comment:
Now it crashes if the kind parameter is missing.

Error: `Cannot read properties of null (reading 'split')`